tests/onnx/test_onnx_v2.py

```diff
@@ -284,6 +284,13 @@ def _onnx_export(self, test_name, name, model_name, feature, onnx_config_class_c
         model_class = FeaturesManager.get_model_class_for_feature(feature)
         config = AutoConfig.from_pretrained(model_name)
         model = model_class.from_config(config)
+
+        # It seems yolo-like models have problems on ONNX exporting on CUDA.
```

Comment:
Suggestion to make the comment more precise:
```suggestion
        # Dynamic axes aren't supported for YOLO-like models. This means they cannot be exported to ONNX on CUDA devices.
```